Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/ThemeHelper.java

```diff
@@ -0,0 +1,125 @@
+package org.dolphinemu.dolphinemu.utils;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.res.Configuration;
+import android.content.res.Resources;
+import android.os.Build;
+
+import org.dolphinemu.dolphinemu.R;
+import org.dolphinemu.dolphinemu.features.settings.model.IntSetting;
+import org.dolphinemu.dolphinemu.ui.main.MainActivity;
+
+public class ThemeHelper
+{
+  public static final String THEME = "theme";
+  public static final String CURRENT_THEME = "current_theme";
+
+  public static final int DEFAULT = 0;
+  public static final int MONET = 1;
+  public static final int MATERIAL_DEFAULT = 2;
+  public static final int GREEN = 3;
+  public static final int PINK = 4;
+
+  public static SharedPreferences mSharedPreferences;
```

Comment:
```suggestion
  private static SharedPreferences mSharedPreferences;
```

Reply:
Preferably also have a comment somewhere in this file about the reason why we're using SharedPreferences. (That is, that Dolphin's usual settings system isn't initialized as early as we would need it to be.)